cmd/todo_list.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+	gitlab "github.com/xanzy/go-gitlab"
+	lab "github.com/zaquestion/lab/internal/gitlab"
+)
+
+var (
+	todoType   string
+	todoNumRet string
+	Type       string
```

Comment:
Type needs to be public as it's passed as a reference to 'opts.Type', doesn't it?

Reply:
Would this work/look better?

´´´
 diff --git a/cmd/todo_list.go b/cmd/todo_list.go
index c7cf8b2..8fa1c5f 100644
--- a/cmd/todo_list.go
+++ b/cmd/todo_list.go
@@ -14,7 +14,7 @@ import (
 var (
        todoType   string
        todoNumRet string
-       Type       string
+       targetType string
 )
 
 var todoListCmd = &cobra.Command{
@@ -54,11 +54,11 @@ func todoList(args []string) ([]*gitlab.Todo, error) {
        var lstr = strings.ToLower(todoType)
        switch {
        case lstr == "mr":
-               Type := "MergeRequest"
-               opts.Type = &Type
+               targetType = "MergeRequest"
+               opts.Type = &targetType
        case lstr == "issue":
-               Type := "Issue"
-               opts.Type = &Type
+               targetType = "Issue"
+               opts.Type = &targetType
        }
 
        return lab.TodoList(opts, num)
´´´

Reply:
> Type needs to be public as it's passed as a reference to 'opts.Type', doesn't it?

Not really, it's only public when you need to access it directly from somewhere else outside that pkg scope, but the reference can be passed regardless. Go doesn't fully block access to it if the programmers really want it. Actually Go uses the terms `exported` and `unexported`, I think it's better since `private` and `public` gives the impression of `security` somehow. Lets keep it in this way.. I'm going to read more about Golang best practices for this specific case (unexported variable references) and change it later in case necessary.

Reply:
Ahh!! actually, I confused myself here. We are discussing two different things here: "global" vs "public", sorry.
In this case it's not required `targetType` to be global, since it's being used in the same function scope `return lab.TodoList(opts, num)`, so the reference will be valid through the whole `lab.TodoList` call.

Reply:
Fair enough. Lemme know, given the latest push, if you are OK with the construction, or if we still need to refactor it
Cheers!

Reply:
Nope.. all good :D